### PR TITLE
[Doc Issue][Update Teams client to MS 365 host application][2880178]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dc7532b9917c2678514fa87b6a99bfcce8472036/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-localization.md) | 84 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6e6247c1-eea7-456f-a4cf-2e5f57aa284b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/520699040f36acd5593f076298c239000494a567/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-localization.md) | 83 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/255c88bb-38b0-4b41-a8be-0eb619d8ebdb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-ypalikila — DISMISSED** [review]
Approved.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d1d262f3899267cdf2d27090006ae283e76eeaf8/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-localization.md) | 83 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6ab763e2-139c-48fe-8204-b41c20739906) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-ypalikila — APPROVED** [review]
Approved.

File: msteams-platform/concepts/build-and-test/apps-localization.md [hunk 1]
```diff
@@ -79,22 +79,21 @@ Following is an example for localization .json:
 }
 ```
 
-You can provide additional .json files with translations of all the user facing strings in your manifest. These files must adhere to the [Localization file JSON schema](../../resources/schema/localization-schema.md) and they must be added to the `localizationInfo` property of your manifest. Each file correlates to a language tag, which the Teams client uses to select the appropriate strings. The language tag takes the form of `<language>-<region>` but you can omit the `<region>` portion to target all regions that support the desired language.
+You can provide additional .json files with translations of all the user facing strings in your manifest. These files must adhere to the [Localization file JSON schema](../../resources/schema/localization-schema.md) and they must be added to the `localizationInfo` property of your manifest. Each file correlates to a language tag, which the Microsoft 365 host application, such as Teams or Outlook, uses it to select the appropriate strings. The language tag takes the form of `<language>-<region>` but you can omit the `<region>` portion to target all regions that support the desired language.
```
Comment (Rick-Kirkham) [inline msteams-platform/concepts/build-and-test/apps-localization.md:82]: grammar mistake

```suggestion
You can provide additional .json files with translations of all the user facing strings in your manifest. These files must adhere to the [Localization file JSON schema](../../resources/schema/localization-schema.md) and they must be added to the `localizationInfo` property of your manifest. Each file correlates to a language tag, which the Microsoft 365 host application, such as Teams or Outlook, uses to select the appropriate strings. The language tag takes the form of `<language>-<region>` but you can omit the `<region>` portion to target all regions that support the desired language.
```

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c81b29ddc921280c3abb9fc89068198a16ecff0b/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-localization.md) | 83 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8abcbef3-ab7e-4090-8634-6a15a3c9c41f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Rick-Kirkham — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4ecacd8e9b87e71bf0d6849b1f32812d090f9ba6/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-localization.md) | 88 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4cc4fe0d-4d2b-44df-840b-704d94b5d657) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c0ffadcf51485527777b7d73f5aefe75c1638af1/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-localization.md) | 88 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a4bdbb70-c341-49b7-9e4e-d891b0079d08) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**